packages/wonder-blocks-testing/src/gql/mock-gql-fetch.ts

```diff
@@ -8,8 +8,11 @@ import type {GqlFetchMockFn, GqlMockOperation} from "./types";
 export const mockGqlFetch = (): GqlFetchMockFn =>
     mockRequester<GqlMockOperation<any, any, any>, any>(
         gqlRequestMatchesMock,
+        // Note that the identation at the start of each line is important.
         (operation, variables, context) =>
             `Operation: ${operation.type} ${operation.id}
-Variables: ${variables == null ? "None" : JSON.stringify(variables, null, 2)}
-Context: ${JSON.stringify(context, null, 2)}`,
+    Variables: ${
+        variables == null ? "None" : JSON.stringify(variables, null, 2)
+    }
+    Context: ${JSON.stringify(context, null, 2)}`,
```

Comment:
**nit:** I didn't add a test for this right now with a view to getting this fixed quickly.  It really should have one so we can catch this more easily in future.

Reply:
NVM, there IS a test for this...but the snapshot got updated instead of the code getting fixed....now I have to update the snapshot in another audit. lol

My bad.

Reply:
Fixed in https://github.com/Khan/wonder-blocks/pull/1822